docs/source/installation.mdx

```diff
@@ -38,8 +40,8 @@ Currently for Windows users, you need to build bitsandbytes from source:
 
 ```bash
 git clone https://github.com/TimDettmers/bitsandbytes.git && cd bitsandbytes/
-cmake -B build -DBUILD_CUDA=ON -S .
-cmake --build build --config Release
+cmake -DCOMPUTE_BACKEND=cuda -S .
+cmake --build . --config Release
 python -m build --wheel
```

Comment:
```suggestion
pip install .
```
For consistency with above

Reply:
I left it as it is since I can't verify this. Can someone with access to a windows VM double check these?

Reply:
Ah yes good point - cc @wkpark 🙏

Reply:
This works for me.

Alternatively with building the wheel, this also works:
```
python -m build --wheel
pip install dist/bitsandbytes-0.43.0.dev0-cp311-cp311-win_amd64.whl
```